internal/server/node/convert.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Contains functions to convert between protobuf and golang struct.
+
+package node
+
+import (
+	"sort"
+
+	pb "github.com/datacommonsorg/mixer/internal/proto"
+	"github.com/datacommonsorg/mixer/internal/server/model"
+)
+
+// Convert pb.GetTriplesResponse to legacy golang struct to make the API response
+// backward compatible.
+func ToLegacyResult(in *pb.GetTriplesResponse) map[string][]*model.Triple {
+	out := map[string][]*model.Triple{}
```

Comment:
nit: "out" might confuse the readers that this means "outEntity", so maybe change the variable name to "result"?

Reply:
Done